test/js/inline-style-property-map.js

```diff
@@ -8,8 +8,44 @@ suite('Inline StylePropertyMap', function() {
     document.body.removeChild(this.element);
   });
 
-  test('Element.styleMap method returns a StylePropertyMap object', function() {
+  test('the Element.styleMap method returns a StylePropertyMap object for that element', function() {
     var inlineStyleMap = this.element.styleMap();
-    assert.instanceOf(inlineStyleMap, StylePropertyMap, 'The styleMap method should return an instance of StylePropertyMap');
+    assert.instanceOf(inlineStyleMap, StylePropertyMap);
+  });
+
+  test('The set method should correctly set a supported StyleValue to a CSS style property of an HTML element', function() {
+    var inlineStyleMap = this.element.styleMap();
+    var simpleLength = new SimpleLength(9.2, 'percent');
+    inlineStyleMap.set('height', simpleLength);
+
+    assert.strictEqual(this.element.style['height'], simpleLength.cssString);
+  });
+
+  test('The set method should correctly set a supported KeywordValue to a CSS style property of an HTML element', function() {
+    var inlineStyleMap = this.element.styleMap();
+    var keyword = new KeywordValue('auto');
+    inlineStyleMap.set('height', keyword);
+
+    assert.strictEqual(this.element.style['height'], keyword.cssString);
```

Comment:
This test is actually testing the same thing as the above test. You can delete it.